src/Components/calculator.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+class Calculator extends React.Component {
```

Comment:
- [ ] This should be a functional component, not a class component. Kindly refactor your code to meet this requirement.

- [ ] This Calculator component should have at least one child component. Kindly do that.